Superposition/ReferenceImplementation.qs

```diff
@@ -175,21 +185,6 @@ namespace Quantum.Kata.Superposition {
         }
     }
 
-    // ------------------------------------------------------
```

Comment:
The whitespace coming back and this task disappearing look like you've created your branch off the master earlier than https://github.com/microsoft/QuantumKatas/commit/debff12623dd85fc4dd2651b0b14c02a0c585a0d. Could you please make sure that your changes are based on the latest state of master branch and don't undo any other changes?